FWCore/ParameterSet/bin/edmWriteConfigs.cpp

```diff
@@ -304,7 +321,11 @@ int main(int argc, char** argv) try {
       std::set<std::string> usedCfiFileNames;
       edm::for_all(pluginNames, std::bind(&writeCfisForPlugin, _1, factory, std::ref(usedCfiFileNames)));
       if (not pluginNames.empty()) {
-        writeModulesFile();
+        bool res = writeModulesFile();
+        if (!res) {
+          std::cerr << "writeModulesFile() failed" << std::endl;
+          return 1;
+        }
```

Comment:
Now this would go back to
```suggestion
        writeModulesFile();
```